sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md

```diff
@@ -0,0 +1,236 @@
+    BinaryData.FromStream(stream),
+    VisualFeatures.Caption,
+    new ImageAnalysisOptions { genderNeutralCaption = true }); // Optional (default is false)
+
+// Print caption results to the console
+Console.WriteLine($"Image analysis results:");
+Console.WriteLine($" Caption:");
+Console.WriteLine($"   '{result.Caption.Text}', Confidence {result.Caption.Confidence:F4}");
+```
+
+### Generate an image caption for an image URL
+
+This example is similar to the above, expect it calls the `Analyze` method and provides a [publicly accessible image URL](https://aka.ms/azai/vision/image-analysis-sample.jpg) instead of a file name.
+
+```C# Snippet:ImageAnalysisGenerateCaptionFromUrl
+// Get a caption for the image. This will be a synchronously (blocking) call.
+ImageAnalysisResult result = client.Analyze(
+    new Uri("https://aka.ms/azai/vision/image-analysis-sample.jpg"),
+    VisualFeatures.Caption,
+    new ImageAnalysisOptions { genderNeutralCaption = true }); // Optional (default is false)
+
+// Print caption results to the console
+Console.WriteLine($"Image analysis results:");
+Console.WriteLine($" Caption:");
+Console.WriteLine($"   '{result.Caption.Text}', Confidence {result.Caption.Confidence:F4}");
+```
+
+### Extract text from an image file
+
+This example demonstrates how to extract printed or hand-written text for the image file [sample.jpg](https://aka.ms/azai/vision/image-analysis-sample.jpg) using the `ImageAnalysisClient`. The synchronous (blocking) `Analyze` method call returns a `ReadResult` object. This object includes a list of text lines and a bounding polygon surrounding each text line. For each line, it also returns a list of words in the text line and a bounding polygon surrounding each word.
+
+```C# Snippet:ImageAnalysisExtractTextFromFile
+// Load image to analyze into a stream
+using FileStream stream = new FileStream("image-analysis-sample.jpg", FileMode.Open);
+
+// Extract text (OCR) from an image stream. This will be a synchronously (blocking) call.
+ImageAnalysisResult result = client.Analyze(
+    BinaryData.FromStream(stream),
+    VisualFeatures.Read);
+
+// Print text (OCR) analysis results to the console
+Console.WriteLine("Image analysis results:");
+Console.WriteLine(" Read:");
+
+foreach (DetectedTextBlock block in result.Read.Blocks)
+    foreach (DetectedTextLine line in block.Lines)
+    {
+        Console.WriteLine($"   Line: '{line.Text}', Bounding Polygon: [{string.Join(" ", line.BoundingPolygon)}]");
+        foreach (DetectedTextWord word in line.Words)
+        {
+            Console.WriteLine($"     Word: '{word.Text}', Confidence {word.Confidence.ToString("#.####")}, Bounding Polygon: [{string.Join(" ", word.BoundingPolygon)}]");
+        }
+    }
+```
+
+### Extract text from an image URL
+
+This example is similar to the above, expect it calls the `Analyze` method and provides a [publicly accessible image URL](https://aka.ms/azai/vision/image-analysis-sample.jpg) instead of a file name.
```

Comment:
nit: I would re-word this to stand alone without reference to the previous snippet